src/main/kotlin/blackjack/domain/card/Cards.kt

```diff
@@ -0,0 +1,12 @@
+package blackjack.domain.card
+
+class Cards(val value: MutableList<Card> = mutableListOf()) {
```

Comment:
해당 value는 외부에서도 수정이 가능하겠죠? 외부에서 수정하지 못하도록 막는 것이 저는 좋다고 생각해요. 외부에서 수정하게 되면 깨진 유리창 처럼 언제든지 사이드 임팩트가 생길 수 있기 때문이죠.

```suggestion
class Cards(value: List<Card>) {

    private val value : MutableList<Card> = value.toMutableList()
```

저 같은 경우에는 이런식으로 변경을 하고, getMethod를 새로 정의할 것 같네요.